app/uk/gov/hmrc/bindingtariffclassification/service/CaseService.scala

```diff
@@ -27,27 +28,37 @@ import scala.concurrent.Future
 
 @Singleton
 class CaseService @Inject()(caseRepository: CaseRepository,
+                            crypto: Crypto,
                             sequenceRepository: SequenceRepository,
                             eventService: EventService) {
 
   def insert(c: Case): Future[Case] = {
-    caseRepository.insert(c)
+    val encryptedCase = crypto.encrypt(c)
```

Comment:
Should we not be doing encryption/decryption in the repository? Seems like its a repository concern to me

Reply:
I did not want to add logic there...